setup.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+packages = [
+    'robottelo',
+]
+
+requires = [
+    'ddt',
+    'fauxfactory',
+    'nailgun',
+    'paramiko',
+    'python-bugzilla',
+    'requests',
+    'selenium',
+]
+
+with open('README.rst', 'r') as f:
+    readme = f.read()
+
+setup(
+    name='robottelo',
+    version='0.1.0',
+    description='Robottelo is a test suite which exercises The Foreman.',
+    long_description=readme,
+    author=u'Satellite QE Team',
+    url='https://github.com/SatelliteQE/robottelo',
+    packages=packages,
```

Comment:
I listed manually because I don't want the tests package to be included. But taking another look on find_packages I have found that we can exclude using a pattern. I have updated the code to use the find_packages, thank you making me revisit that is aways good to learn new things :smile: